src/components/Menu/MenuItemButton/MenuItemButton.scss

```diff
@@ -10,7 +10,7 @@
   padding: $spacing-xs 0;
   align-items: center;
 
-  .monday-style-button {
+  .menu-item-button {
```

Comment:
there is potential for bugs here because I think monday-style-button used to be the class name of the button component. We need to compare the appearance and check if after the change there is not styles which now missing from the menu item.